src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -301,6 +301,7 @@ export function PostDialogUI(props: PostDialogUIProps) {
             {!process.env.STORYBOOK && (
                 <RedPacketDialog
                     classes={classes}
+                    theme={props.theme}
```

Comment:
It makes sense to set `<RedPacketDialog/>` the same theme of its parent `<PostDialog />` .